management/models.py

```diff
@@ -34,10 +35,10 @@ class PermissionsBase(models.Model):
         max_length=8, choices=PERMISSIONS_LEVELS, default="private"
     )
 
-    def set_permissions(self):
-        """Set object-level permissions."""
+    def set_object_permissions(self):
+        """Set object-level delete, change and view permissions."""
 
-        delete, change, view = _get_perm_codenames(self.__class__)
+        delete, change, view, add = _get_perm_codenames(self.__class__)
```

Comment:
The `add` permission is not used in this function. Should it be used somewhere?

Reply:
The `add` permission is used in the `set_model_permissions` function below this one